.github/workflows/test.yml

```diff
@@ -3,12 +3,13 @@ name: Tests
 on:
   push:
   pull_request:
-  schedule:
-    - cron: '0 6 * * *'
```

Comment:
why?   It is good to have things tested regularly to make sure that it still all works eg. when new pydantic comes out etc.
 
I haven't spotted any commit message summary among 22 (so didn't go one by one) to see if more details would be given.

NB filing as a separate comment to not await for my full review

Reply:
it seems to be solving a different problem. it would be good to use less electrons when possible. once we get into lazy mode with schema, we can turn cron job back on. or if we did this once a week or month. every 6 hours is total overkill.

Reply:
That's not "every 6 hours"; it's "once a day at 6 AM".

Reply:
that is true.

Reply:
pushed bringing back daily run - the change was OOS for this PR ;)